blockdata/blockdata.go

```diff
@@ -91,7 +91,7 @@ func (b *BlockData) ToBlockExplorerSummary() apitypes.BlockExplorerBasic {
 		FreshStake:             b.Header.FreshStake,
 		StakeDiff:              b.Header.SBits,
 		BlockExplorerExtraInfo: extra,
-		Time:                   b.Header.Time,
+		Time: b.Header.Time,
```

Comment:
please undo the go fmt changes as we'll just keep going back and forth on this.  We're ignoring gofmt warnings in CI for now, so it's fine.